---
title: "Marie-Curie post-doc proposal"
format:
  docx:
    reference-doc: reference.docx
editor: source
---

# Part B-1 (maximum of 10 pages, Times New Roman, font-size = 11)

## 1.  Excellence

### 1.1 Quality and pertinence of the project’s research and innovation objectives (and the extent to which they are ambitious, and go beyond the state of the art)

One of the biggest threats to Europe’s biodiversity are invasive alien species (IAS). Europe’s societies are driven by trade and transport from across the globe. This causes organisms to be moved to novel areas where they can establish and become invasive (i.e. cause substantial harm to biodiversity and/or ecosystem services). Protected areas (such as the Natura 2000 network) were introduced to protect Europe’s native biodiversity and are, in many cases, refuges for threatened species that are vulnerable to the threat of IAS. However, this means that it is crucial that these protected areas remain free of IAS.

Controlling and/or stopping the spread of IAS is challenging and, in many cases, impossible once a species becomes established. However, decades of research has shown that preventing invasions is possible if species are detected early enough. Environmental DNA (eDNA: DNA present in water, in soil or even in the air) methods are highly sensitive and can detect species when they are at very low abundance (i.e. before they become established). Therefore, a surveillance system using eDNA methods could be a useful tool for catching prospective IAS early and preventing their spread into Europe’s protected areas.

Environmental monitoring always faces the problem of resource limitations. As such, an eDNA-based surveillance system would need to balance resource constraints and be able to collect information in a way that will provide the most possible information about the risk of prospective IAS to protected areas. In this project, using Belgium as a test-case, I will develop an eDNA-based IAS surveillance system that uses information theory and Bayesian modeling to identify sets of sampling sites that will optimally allocate limiting sampling resources to maximise information about the risk of prospective IAS to protected areas. This system can then be used to guide IAS management or further sampling.

I call this project AIRIES (Adaptive Invasion-Risk Intelligence for Protected Areas with eDNA Surveillance) and it is structured around **four measurable, verifiable and achievable R&I objectives**:

**Objective 1:** Model the risk of 50 prospective IAS (i.e. species that are not yet present in Belgium or have a very isolated range) to the Natura 2000 network of Belgium using environmental data, dispersal kernels and expert assessments of species’ potential impact.

+ **Target deliverable 1:** Publicly available spatial risk maps for the 50 prospective IAS (GeoTIFF files) and an interactive web dashboard where these risk maps can be visualised and queried.

**Objective 2:** Develop an information theory algorithm that quantifies how much additional information about the risk of IAS to the Natura 2000 is obtained by taking eDNA samples in a specific location. Use this algorithm to identify priority for sites for eDNA sampling using optimisation modelling.

+ **Target deliverable 2:** Information theory algorithm and a list of locations in Belgium where new information will improve our ability to detect IAS in the Natura 2000 network. This tool will be integrated to the interactive web dashboard from objective 1.

**Objective 3:** Build a metabarcoding reference database for these 50 prospective IAS using genetic databases and targeted sampling.

+ **Target deliverable 3:** Curated and publically available metabarcoding reference database for these prospective IAS in Belgium for use in DNA metabarcoding.

**Objective 4:** Collect eDNA samples at these priority locations, perform metabarcoding and taxonomic assignment using the reference database developed in **objective 2** and develop a data ingestion pipeline that can feed back into the information theory algorithm developed in **objective 3**.

+ **Target deliverable 4:** eDNA samples from priority sites in Belgium identified in **objective 3** and an update to the interactive web dashboard to allow the ingestion of samples from the priority, risk updating and priority site updating.

These four objectives will be addressed as four complementary **work packages (1-4)**. Given my background in biomonitoring, modeling and data science, these objectives are achievable and progress towards them can be clearly measured using the identified **target deliverables (1-4)**. By completing these four objectives, AIRIES would become the first information-theory driven adaptive IAS surveillance system and it would be specifically tailored to safeguard Europe’s protected area network from the increasing threat of IAS. Therefore, not only will AIRIES deliver novel research, it will deliver a working prototype of a surveillance system that, if successful, could be applied to IAS management across Europe.

### 1.2 Soundness of the proposed methodology (including interdisciplinary approaches, consideration of the gender dimension and other diversity aspects if relevant for the research project, and the quality of open science practices).

The AIRIES project focuses on 50 prospective plant and mammal IAS in Belgium. These include IAS that experts deem likely to reach Belgium (e.g. grey squirrel: *Sciurus carolinensis*), IAS that are present in neighbouring countries but have not established in Belgium (e.g. the coastal succulent: *Carpobrotus edulis*) or IAS that are present in Belgium but have restricted ranges (e.g. the North American shrub: *Baccharis halimifolia*).

#### **WP1**: Modeling the risk of new IAS in Belgium

We start by rasterising Belgium into 1 x 1 km grid-cells (*ca.* 30000) using the ETRS89 / LAEA Europe Lambert Azimuthal Equal Area projection (EPSG:3035). The risk framework used in AIRIES defines the risk of a given IAS in grid-cell $i$ to protected area p as:

$$
R_{i,p} = \psi_i  \cdot w_{species} \cdot w_{site} \cdot K_{i,p}
$$

Where:

$\psi_i$ is the probability that a focal species is present in cell $i$ [0, 1]
$w_{species}$ is a weight describing the potential ecological damage of the species [0, 1]
$w_{site}$ is a weight describing the ecological value of the protected area [0, 1]
$K_{i,p}$ is the dispersal kernel describing the connectivity between grid-cell $i$ and protected area $p$ [0, 1].

The $K_{i,p}$ dispersal kernel is an exponential kernel defined as follows where $d(i,p)$ is the distance between grid-cell $i$ and protected area $p$ and $\alpha$ is the mean dispersal distance of the species.

$$
K_{i,p} = exp(-d(i, p)/\alpha)
$$

This means that the risk of a given IAS from grid-cell $i$ to protected area $p$ is directly proportional to the probability that the IAS is present in grid-cell $i$ ($\psi_i$), the potential ecological damage of the species ($w_{species}$), the ecological value of the protected area ($w_{site}$) and how connected grid-cell $i$ is to protected area $p$. With this framework, we can obtain, for each IAS and protected area $p$, risk scores for each grid-cell ($R_{i,p}$) of Belgium.

To estimate the $\psi_i$ (i.e. probability of occurrence of a given IAS in grid-cell $i$), we will use habitat suitability models fit in a Bayesian framework. These models will be trained on presence-absence (or pseudo-absence) data of the chosen IAS from their current range. Using this model, we will obtain samples from the posterior distribution of $\psi_i$ i.e. $\psi_i^{(s)}$, where the $^{(s)}$ indicates samples. Therefore, the distribution of values for a given grid-cell ($\psi_i^{(s)}$) represents our prior belief about where the IAS is likely to be present based on its current distribution and environmental preferences.

The $w_{species}$ values will be based on expert assessments of IAS impacts via the Harmonia+ protocol. This protocol asks experts to rank, for example, environmental risk of a species on a five point scale: very low, low, medium, high, very high. Based on a number of similar questions about the risk a species poses, these are converted into a number between 0 and 1 and the scores are then aggregated to provide an overall impact score. To derive the $w_{site}$ values, we use the Standard Data Form for Natura 2000 sites to derive an ecological value index for each protected area ($w_{site}$). The $K_{i,p}$ dispersal kernel will be based on the distance between the centroid of grid-cell i and the centroid of protected area p. Where possible, average dispersal distances will be used.

Using the $\psi_i^{(s)}$, $w_{species}$, $w_{site}$ and $K_{i,p}$, we will calculate the risk-score values for each grid-cell $i$ and protected area $p$ (i.e. $R_{i,p}$) for all of the focal IAS. These risk-scores will be based on all samples of $\psi_i^{(s)}$ and, as a result, for each grid-cell $i$ and protected area $p$, we will have a distribution of risk-score values: $R_{i,p}^{(s)}$. The risk-score maps per species and protected area will then be published as GeoTIFF files along with all code used for the analysis. In addition, we will wrap these risk-score maps into an interactive web application using RShiny that will allow various stakeholders (e.g. researchers, civil society, interested citizens etc.) to engage with the data and examine the risk of each IAS to a given protected area (i.e. **target deliverable 1**).

#### **WP2**: Developing an information theory algorithm for optimal eDNA sampling

In **WP1**, we will create maps of prior risk scores (i.e. $R_{i,p}^{(s)}$) for each IAS for each protected area $p$ from each grid-cell $i$. The next step in AIRIES is to determine the optimal locations for eDNA sampling that will maximise the amount of new information about the risk of a given IAS to protected area $p$ which is the focus of **WP2*. 

At a general level, change in information can be quantified using the Kullback-Leibler divergence, a measure of the discrepancy between two probability distributions that quantifies the expected information gain when one revises their beliefs from a prior distribution to a posterior distribution. In this case, we can view our prior belief about the risk of a given IAS to protected area $p$ from grid-cell $i$ as $R_{i,p}^{(s)}$ which we now write as: $R_{i,p,\text{prior}}^{(s)}$. We then update this belief by taking and analysing eDNA samples which gives a new distribution of risk scores: $R_{i,p,\text{updated}}^{(s)}$. We can then compute the Kullback-Leibler divergence between these distributions to calculate the change in information about the risk from a given IAS to protected area $p$ from taking and analysing an eDNA sample in a given grid-cell $i$.

To estimate $R_{i,p,\text{updated}}^{(s)}$ (i.e. the updated risk-score following eDNA sampling and analysis), requires simulating the likely information we will obtain from eDNA sampling and analysis. This requires an observational model of how the presence or absence of a given IAS along with measurement error determines eDNA detections. We propose the following observation model where eDNA detections ($y_ij$) in sample ($j$) and grid-cell ($i$) translate to the probability of detection ($p_i$) as:

$$
y_{ij} \sim Bernoulli(p_{i})
$$

$$
p_{i} = z_i \times (1 - \theta^{FN}_{field}) \times (1 - \theta^{FN}_{lab}) + (1 - z_i) \times \theta^{FP}_{lab}
$$

Where:

$z_i$: true presence (1) or true absence (0)

$\theta^{FN}_{field}$ false negative rate in the field (i.e. non-detection when an IAS is present). Therefore, 1 - $\theta^{FN}_{field}$ is the true positive rate in the field (i.e. the probability that an IAS is detected when it is present).

$\theta^{FN}_{lab}$ false negative rate in the lab (i.e. non-detection when an IAS is present in the sample). Therefore, 1 - $\theta^{FN}_{lab}$ is the true positive rate in the lab (i.e. the probability that an IAS is detected following PCR when its DNA is present).

$\theta^{FP}_{lab}$ false positive rate in the lab (i.e. detection when an IAS is absent from the sample which can occur via, for example, contamination).

With this as our underlying observational process model, we can use Bayesian updating to determine how the prior probability of occurrence estimated in **WP1** (i.e. $\psi_i$ which we now denote as $\psi_{i,prior}$) and, therefore, the risk-score would change if we sampled eDNA. For this, we use Bayes’ rule. More specifically, assume that we sample eDNA in grid-cell $i$ and obtain a positive detection for species A (i.e. $y_i = 1$). We calculate the posterior probability that species A is present (i.e. $z_i = 1$) in grid-cell $i$ using Bayes’ rule as follows:

$$
P(z_i=1|y_i = 1) = \frac{p_{11} \psi_{i,prior}}{p_{11} \psi_{i,prior} + p_{01}(1-\psi_{i,prior})}
$$

Where:

$z_i \epsilon [0,1]$: true presence (1) or absence (0) of species A in grid-cell $i$ $y_i \epsilon [0,1]$: detection (1) or non-detection (0) of species A in grid-cell $i$

$\psi_{i,prior} = P(z_i = 1)$: prior probability that species A is present in grid-cell $i$

$p_{11} = P(y_i = 1|z_i = 1)P(z_i = 1) = (1 - \theta^{FN}_{field}) \cdot (1 - \theta^{FN}_{lab})$: true positive rate derived from the probability of false negatives in the field and lab.

$p_{01} = P(y_i = 1|z_i = 0)P(z_i = 0) = \theta^{FP}_{lab}$: false positive rate derived from the probability of false positives in the lab

However, there is also the possibility that the eDNA sampling results in non-detection (i.e. $y_i = 0$). We need to account for both possibilities. For this, we need to calculate the posterior probability that species A is present (i.e. $z_i = 1$) in grid-cell $s$ given non-detection (i.e. $y_i = 0$):

$$
P(z_s = 1|y_s = 0) = \frac{(1 - p_{11}) \psi_{i,prior}}{(1 - p_{11}) \psi_{i,prior} + (1 - p_{01})(1-\psi_{i,prior})}
$$

Using these simulated probabilities of occurrence for a given grid-cell $i$ under detection ($P(z_i=1|y_i = 1)$) and no detection ($P(z_i=1|y_i = 0)$) scenarios, we can calculate updated risk scores.

Assuming detection, we get the following:

$$
P(z_i=1|y_i = 1) \sim \psi_{i, \text{ updated: detection}}^{(s)}
$$

$$
R_{i,p, \text{ updated: detection}}^{(s)} = \psi_{i, \text{ updated: detection}}^{(s)} \cdot w_{species} \cdot w_{site} \cdot K_{i,p}
$$

Assuming non-detection, we get the following:

$$
P(z_i=1|y_i = 0) \sim \psi_{i, \text{ updated: non-detection}}^{(s)}
$$

$$
R_{i,p,\text{ updated: non-detection}}^{(s)} = \psi_{i, \text{ updated: non-detection}}^{(s)} \cdot w_{species} \cdot w_{site} \cdot K_{i,p}
$$

With samples from the posterior distribution of the risk to protected area $p$ from grid-cell $i$ under scenarios of detection ($R_{i,p,\text{ updated: detection}}^{(s)}$) and non-detection ($R_{i,p,\text{ updated: non-detection}}^{(s)}$), we can compute the KL-divergence under both these scenarios. Note that $\hat{f}$ denotes the kernel density estimated distribution:

$$
D_{KL, \text{ detection}} = \mathbb{E}\biggl[log\frac{\hat{f}(R_{i,p,\text{ updated: detection}}^{(s)})}{\hat{f}(R_{i,p, \text{ prior}}^{(s)})}\biggr]
$$

$$
D_{KL, \text{ non-detection}} = \mathbb{E}\biggl[log\frac{\hat{f}(R_{i,p, \text{ updated: non-detection}}^{(s)})}{\hat{f}(R_{i,p,\text{ prior}}^{(s)})}\biggr]
$$

The **Expected Information Gain** in grid-cell $i$ to the protected area $p$ ($EIG_i,p$) from sampling is then a weighted average of these two KL-divergences which is weighted by the probability of detection ($p_{det}$). The probability of detection is given by:

$$
p_{det} = P(y_s = 1) = p_{11}\psi + p_{01}(1 - \psi)
$$

Because $p_{det}$ is a distribution of samples (i.e. $p_{det}^{(s)}$), this results in a distribution of $EIG_i,p$ values i.e. ($EIG_{i,p}^{s}$):

$$
EIG_{i,p}^{s} = p_{det}^{(s)}\cdot D_{KL, \text{ detection}}  + (1 - p_{det}^{(s)}) \cdot D_{KL, \text{ non-detection}}
$$

This example above is based on one IAS and one protected area. However, the idea behind **AIRIES** is to do this for 50 different invasive alien species across the Natura 2000 network of Belgium. Therefore, we will calculate **Expected Information Gain** for each grid-cell, for each IAS and each protected area. We will then use optimisation modeling to prioritise sites that provide maximum information value across species and protected areas given that, due to resource limitation, we can only conduct eDNA sampling in a limited number of grid-cells. More specifically, our objective is to select a limited number of grid-cells to sample that, together, provide the greatest **Expected Information Gain** across multiple species and protected areas. Formally:

$I$: the set of candidate grid cells
$C$ the set of all species--protected area combinations
$P$: the set of protected areas
$EIG_{ic}$: the expected information gain (**EIG**) for grid cell $i$ in combination $c$.

Next, we define two sets of binary decision variables:

$y_i$ = 1 if grid cell $i \in I$ is selected; 0 otherwise
$z_c$ = 1 ) if species-protected area combination $c \in C$ is covered by at least one selected grid-cell; 0 otherwise

Our objective is then to maximise:

$$
\text{Maximise} \sum_{c \in C} \sum_{i \in I_c} EIG_{ic} \cdot y_i
$$

Where ( $I_c \subseteq I$ ) is subset of grid-cells for combination $c$ where the *EIG* value is greater than the 80% quantile. This optimisation algorithm therefore attempts to select grid-cells that contribute the **highest possible cumulative information gain** across species and protected areas. Additionally, we set the following constraints to make sure that we sample in a way that balances protected area coverage:

**1. Sample size constraint:** Select exactly $X$ grid-cells:

$$
\sum_{i \in I} y_i = X
$$

**2. Combination coverage constraint:** A species by protected area combination is considered "covered" if **at least one** of its associated top-EIG grid cells is selected:

$$
z_c \leq \sum_{i \in I_c} y_i \quad \text{ for all } c \in C
$$

**3. Protected area coverage constraint:** At least one species by protected area combination must be covered for **every protected area** ($p \in P$) where $C_p \subseteq C$ is the set of combinations associated with protected area $p$:

$$
\sum_{c \in C_p} z_c \geq 1 \quad \text{ for all } p \in P
$$

Using our risk framework, information theory and optimisation modelling, we will be able to prioritise sites for eDNA sampling that will provide the maximum amount of information about the risk of Belgium's protected area network to prospective IAS. This information theory approach and optimisation algorithm will be integrated into the interactive RShiny web application (i.e. from **target deliverable 1**) which will allow stakeholders to see where our approach suggests eDNA sampling given a set number of possible grid-cells to sample. Together, the development of the approach and the integration with the interactive web dashboard constitute **target deliverable 2**.

::: {#fig-1}
![](figures-tables/conceptual-figures/fig-1.png)

Overview of the **AIRIES** information theory and optimisation algorithm for selecting sites for eDNA sampling and analysis that maximise information gain about the risk of IAS to Belgium’s protected area network. This figure relates to ****WP1**** and ****WP2***.
:::

#### **WP3**: Building a metabarcoding reference database for the 50 prospective IAS

Given that, in **WP2*, we will establish a list of priority sites for eDNA sampling that will provide the maximum amount of information about the risk of the 50 IAS to Belgium's protected areas, the next step is the eDNA sampling. Detecting species using eDNA is typically done using one of two approaches: (1) **quantitative PCR (i.e. qPCR)** or (2) DNA metabarcoding (**metabarcoding**). If one is interested in detecting whether a specific species is present or absent given an eDNA sample, qPCR is typically favoured. However, qPCR requires a separate assay to be developed for each species of interest. In contrast, a more general approach is metabarcoding whereby a short, taxonomically informative DNA region that is flanked by two conservative regions (i.e. primer-pairs) is amplified and sequenced and the sequences are compared against a metabarcoding reference database for taxonomic assignment. The reference database is a set of reference sequences for the target taxa that the obtained sequences can be compared against. In this project, we are aiming to monitor 50 priority IAS and, therefore, we will use metabarcoding.

Given the established priority sites identified in **WP2* for eDNA sampling to maximize detection efficiency of 50 IAS threatening Belgium’s protected areas, **WP3** will focus on building a high-quality metabarcoding reference database. This database will substantially enhance the accuracy of species detection from eDNA samples, reducing false-positive matches during taxonomic assignment.

To detect our focal IAS, comprising mammals and plants, we will use established primer sets known for their discriminatory power. Mammals will be targeted using the mitochondrial 16S rRNA primers, amplifying an approximately 95 bp fragment. Plants will be targeted using two primer sets, the nuclear ITS2 primer set and the and plastid rbcl primer regions. Initially, comprehensive searches for publicly available barcode sequences will be conducted across global genetic repositories including the European Nucleotide Archive (ENA), GenBank, DNA Data Bank of Japan (DDBJ), and Barcode of Life Data Systems (BOLD). Queries will incorporate known synonyms derived from authoritative global taxonomic resources such as the Global Biodiversity Information Facility (GBIF), Integrated Taxonomic Information System (ITIS), and Catalogue of Life.

Following initial sequence compilation, a structured gap analysis will identify IAS lacking sufficient barcode coverage. To address identified gaps, targeted specimen collection campaigns will be organized in collaboration with key biological collections such as Meise Botanic Garden and the Royal Belgian Institute of Natural Sciences. Specimen verification through taxonomic experts will precede genetic analysis. Specimen DNA extraction protocols will follow best practices optimized for both contemporary and archival samples, prioritizing yield and DNA integrity. PCR amplification using species-specific primer sets will follow standardized protocols. Resulting PCR products will undergo Sanger sequencing to generate high-quality barcode sequences.

All obtained sequences will undergo stringent bioinformatics quality control analogous to established reference database workflows. Steps will include:

1. Removal of sequences with incomplete or ambiguous taxonomic annotations.

2. Filtering for potential contamination through BLAST comparisons against representative sequences from common contaminants (e.g., Homo sapiens, microbial symbionts).

3. Alignment using MAFFT to ensure sequence homology, discarding sequences causing alignment gaps indicative of errors or contamination.

4. Trimming sequences to standardized barcode lengths (e.g., 95 bp for mammals; ITS2 and rbcl standard lengths for plants) using R packages such as 'spider'.

5. Selection of representative haplotypes per species with the R package 'haplotypes' to eliminate redundancy and enhance database efficiency.

The finalized metabarcoding reference database will be formatted for compatibility with established bioinformatics pipelines (e.g., QIIME2) to ensure broad usability and reproducibility and published on a publicly available, data-sharing platform like Figshare, Dryad or Zenodo. Regular updates to the database will be made publicly available, reflecting ongoing data curation and addition of new validated sequences. Whilst this reference database is tailored towards 50 priority IAS for Belgium, these IAS overlap with other European countries and, as aresult, this reference database will likely be broadly applicable for IAS monitoring. This reference database and its publication constitute **target deliverable 3**.

#### **WP4**: Targeted, multi-substrate eDNA sampling of IAS and risk updating

With the list of priority sampling sites based on **WP1** and **WP2** along with the metabarcoding reference database from **WP3**, the step in the AIRIES project is to conduct eDNA sampling in these priority sites and use the information gathered from this eDNA sampling to update the risk maps. To do this, we will employ multi-substrate eDNA sampling of air, water and soil.

Recent work has shown that different eDNA substrates—air, water, and soil—capture complementary signals from biodiversity present in the landscape. To maximise detection probability of priority invasive alien species (IAS) across environmental gradients and life-history traits, the AIRIAS project will implement a harmonised multi-substrate sampling strategy. By integrating signals from these three substrates at each priority site, we aim to reduce false negatives and provide a more robust estimate of IAS presence.
At each site, samples from air, water, and soil will be collected within the same 24-hour window to ensure temporal comparability. Four replicate air samples will be taken over a 12-day period and pooled per site to represent a composite airborne eDNA signal. Water samples will be collected from suitable aquatic features (e.g. ponds, ditches, or streams) using sterile bottles, with multiple subsamples pooled to reduce spatial heterogeneity. For soil, surface material (0–5 cm depth) will be collected from multiple locations within each site and homogenised into a composite sample.

DNA extraction will follow substrate-specific protocols optimised for yield and inhibitor removal. Air filter material will be processed using a modified DNeasy Blood & Tissue Kit protocol, while water samples will be filtered on-site or in the lab using 0.45 μm filters, followed by extraction with the same kit. Soil samples will be extracted using the DNeasy PowerSoil Kit (Qiagen) to address high humic acid content. All extractions will be conducted in a clean-room facility with strict contamination control procedures, including the use of dedicated equipment, UV sterilisation, and PPE.

Amplification of target loci will proceed via a two-step PCR metabarcoding workflow. For mammals, a ~95 bp fragment of the mitochondrial 16S rRNA gene will be amplified using the 16Smam1 and 16Smam2 primers, with a human-blocking primer to reduce non-target amplification. For plants, two loci will be targeted: the nuclear ITS2 region (using ITS2-S2F and ITS4 primers) and the plastid rbcl region. PCRs will be run in triplicate for each substrate-sample combination, and amplicons will be pooled and cleaned using magnetic bead-based purification. In the second PCR step, dual-index barcodes and sequencing adaptors will be added.

Sequencing libraries will be quantified, pooled equimolarly, and sequenced using 2×300 bp paired-end chemistry on the AVITI platform (Element Biosciences). Amplicon Sequence Variants (ASVs) will be inferred using the DADA2 pipeline, and taxonomic assignments will be made via BLAST searches against the curated, Belgium-focused reference database developed in WP3. Only high-confidence matches to one of the 50 target IAS will be retained, and all results will be interpreted on a presence–absence basis.

The resulting species detection data across substrates will be used to update spatial risk maps of IAS occurrence. Comparative analyses will assess congruence and complementarity between substrates, providing insight into optimal substrate combinations and guiding future surveillance design in protected areas.

#### Interdisciplinarity

AIRIES is fundamentally interdisciplinary. It integrates cutting-edge ecological modelling, Bayesian statistical methods, information theory, advanced genetic methodologies (i.e. eDNA metabarcoding), optimisation algorithms, and software development. This will not only improve our predictive understanding of IAS but it will also deliver a novel, adaptive surveillance system that is...

By bridging these distinct disciplines, AIRIES not only enhances our predictive understanding of invasive species risks but also delivers a novel, adaptive surveillance system uniquely suited to real-world management needs. This blend of quantitative ecology, genetic surveillance technology, and applied informatics represents a significant innovation, positioning AIRIES at the forefront of invasion biology and biodiversity conservation.

#### Open science to the core

**AIRIES** is, at its core an open science project. There are varying views about what constitutes *open science*. However, in my view, *open science* has has three key principles: 

*1. All research materials should be publicaly available and constantly shared. This includes data, code, documentation and other research materials.* 

+ The AIRIES project and any sub-projects will be hosted in publicly available Github repositories where all research materials will be open, documented and regularly updated.

*2. Research findings should be available as both open-access scientific publications (e.g. via pre-prints or initiatives such as *Peer Community Journal*) and as accessible content for a general audience (e.g. popular science articles, infographics, dashboards etc.).* 

- For AIRIES, we envision an open-access publication for each work-package (e.g. published in *Peer Community Journal*, *Methods in Ecology and Evolution* etc.). In addition, we will commit to creating and maintaining an interactive web dashboard that integrates information across all work-packages and makes that information available to the general public.

*3. Collaboration with target end-users.*

- I developed the AIRIES project to improve IAS management and safeguard Europe's protected areas. Given that I will use Belgium as a test case, the potential end-users of this research are the nature management organisations in Belgium (e.g. Instituut Natuur en Bosonderzoek, local governments etc.). I will make sure that these organisations are aware of and can access the information associated with AIRIES (e.g. via the interactive web dashboard). In addition, I will make myself available to help them implement the AIRIES system if it fits into their management workflows.


### 1.3 Quality of the supervision, training and of the two-way transfer of knowledge between the researcher and the host

Prof. Bram Vanschoenwinkel, the main supervisor, initiated the **Barcoding for Chocolate** project which aims to use DNA barcoding to identify and understand cacao pollinators. In addition, he is an active collaborator on the **International Barcode of Life** project. Therefore, he has considerable experience with DNA barcoding and metabarcoding techniques that are highly relevant for this project. In addition, Prof. Vanschoenwinkel is actively involved in biomonitoring projects for the management of invasive species. More specifically, he has been designing novel LiDAR based sensors for tracking insects including some invasive insects in Belgium. Therefore, he also has experience with applied biomonitoring systems which is the focus of this project. 

Sophie de Buyl will be the support person for the modeling part of the project. She is a physicist by training but now works with biological systems.

Dr. Fabian Roger, is the founder of DNAir, a start-up that focuses on developing airborne eDNA as a large-scale biomonitoring tool. A professional placement is envisaged at DNAir where I will gain experience with what it takes to apply biomonitoring and eDNA techniques in a commercial environment. Moreover, DNAir's research and development team is currently working on methods to predict the optimal temporal windows for airborne eDNA sampling to maximise species detection. With the learnings gained in the AIRIES project on information theory, Bayesian modeling and optimisation algorithms, I am sure that I will be able to contribute to their progress in this area.





### 1.4 Quality and appropriateness of the researcher’s professional experience, competences and skills

James Hagan trained as an ecologist and biodiversity scientist. His PhD, at the University of Gothenburg, Sweden centered around the links between biodiversity and ecosystem services and on developing analytical methods to quantify these links. It was during this time that he revisited his love for mathematics and statistics and developed into a skilled statistical modeler and programmer. This resulted in several high profile publications (e.g. Ecology Letters, Nature Ecology and Evolution) and an open source R-package.

With these statistical modeling and programming skills, following his PhD, he spent a year in industry as a data scientist for a public-policy consultancy. As a result, he furthered these skills and gained important knowledge about production-ready data pipelines and the latest data science tools that are used in industry.

Given his training as an ecologist along with his industry-standard data science skills, James now wants to take these skills back to academia and develop the AIRIES project.



## 2. Impact

### 2.1 Credibility of the measures to enhance the career perspectives and employability of the researcher and contribution to his/her skills development

With this post-doc James Hagan will have the opportunity to continue developing his statistical modeling and programming skills. These are skills that will make highly employable as both are strongly in demand in many sectors. Moreover, the planned professional placement at DNAir will give him experience in a commercial biomonitoring environment. This is important if he decides to move into industry or start his own biomonitoring company.

### 2.2 Suitability and quality of the measures to maximise expected outcomes and impacts, as set out in the dissemination and exploitation plan, including communication activities

Scientific publications

Blog posts

Interactive web dashboard

Outreach with local managers. Give seminars at INBO, and the major provincial authorities of Belgium.



### 2.3. The magnitude and importance of the project’s contribution to the expected scientific, societal and economic impacts

IAS have enormous economic and societal costs. It is estimated that Europe spends billions of euros on the management of IAS. The societal costs of IAS are also crucially important as IAS degrade landscapes and, in some cases, have direct impacts on people's quality of life (e.g. Asian hornet). What AIRIES aims to do is develop a surveillance system that can detect IAS before they become established and are able to cause damage. If AIRIES is successful, we could save billions of euros by preventing new IAS from establishing in Europe and, therefore, avoid some of the societal costs that IAS brings.

However, the importance of the project is not only economic and societal. With the AIRIES project, we aim to show how concepts that are more traditionally applied in physics like information theory, optimisation algorithms and Bayesian observation models can be used to prioritise sites for biomonitoring. Biomonitoring always faces the problem of resource constraints. Therefore, a robust framework for prioritising sites that is rooted in the concept of information would be an important scientific contribution that could change the way that scientists and nature managers think about using limited biomonitoring resources.


## 3. Quality and Efficiency of the Implementation


### 3.1 Quality and effectiveness of the work plan, assessment of risks and appropriateness of the effort assigned to work packages


### 3.2 Quality and capacity of the host institutions and participating organisations, including hosting arrangements





